### PR TITLE
Debug symbols

### PR DESCRIPTION
**Pull request by srubin**

### CODE REVIEW

File: build-ffmpeg [hunk 1]
```diff
@@ -679,9 +678,43 @@ download "https://ffmpeg.org/releases/ffmpeg-4.3.1.tar.bz2"
...
 
 execute make -j $MJOBS
+
+if is_mac; then
+	execute dsymutil libavcodec/libavcodec.dylib -o libavcodec.dSYM
+	execute dsymutil libavdevice/libavdevice.dylib -o libavdevice.dSYM
+	execute dsymutil libavfilter/libavfilter.dylib -o libavfilter.dSYM
+	execute dsymutil libavformat/libavformat.dylib -o libavformat.dSYM
+	# we're not building this one
+	# execute dsymutil libavresample/libavresample.dylib -o libavresample.dSYM
```
Comment (marcello3d) [inline build-ffmpeg:693]: don't we need libavresample to resample things? or is it bundled into something else?

File: build-ffmpeg [hunk 2]
```diff
@@ -679,9 +678,43 @@ download "https://ffmpeg.org/releases/ffmpeg-4.3.1.tar.bz2"
...
+	execute dsymutil libavutil/libavutil.dylib -o libavutil.dSYM
+	execute dsymutil libpostproc/libpostproc.dylib -o libpostproc.dSYM
+	execute dsymutil libswresample/libswresample.dylib -o libswresample.dSYM
+	execute dsymutil libswscale/libswscale.dylib -o libswscale.dSYM
+fi
+
+if [[ -n "$SENTRY_AUTH_TOKEN" ]]; then
+	# TODO: install sentry-cli when running on CI
+	sentry-cli --auth-token $SENTRY_AUTH_TOKEN upload-dif --org descript-inc --project descript-electron .
```
Comment (marcello3d) [inline build-ffmpeg:702]: is there a way we can include these dSYM files somewhere so this repo doesn't need to do the uploading itself?

**Review by marcello3d — APPROVED** [review]

File: build-ffmpeg (hunk 1 above)
Comment (srubin) [reply build-ffmpeg:693]: https://github.com/performous/performous/issues/396

No, it uses `libswresample`

File: build-ffmpeg (hunk 2 above)
Comment (srubin) [reply build-ffmpeg:702]: 1. I guess the question is: where would we want to do that? None of the options really "make sense" aside from in beachcube itself. But it seems weird to carry all of the dSYMs from this twice-removed library all the way to that repo.
2. We're uploading these assets (dSYMS + dylibs) to Sentry before stripping them. The dSYMs and dylibs actually have different debug properties (sentry-cli reports that one has `symtab` and `unwind` while the other has `symtab` and `debug`) so we want both (pre-stripping). So we'd have to publish the unstripped binaries somewhere if we didn't want to upload here.

Comment (marcello3d) [reply build-ffmpeg:702]: makes sense. then let's make the org and project environment variables then, so it's not descript-specific logic

File: build-ffmpeg (hunk 1 above)
Comment (marcello3d) [reply build-ffmpeg:693]: ah, maybe add that to the comment?

**Pull request merged**